holidayextras/ui-toolkit
# Adding react-a11y and fixing known issues

#### What does this PR do? (please provide any background)
We really need to concentrate on creating beautiful user experience for everyone.
A11y is a measure of a computer system's accessibility to all people, including those with disabilities or impairments. React-a11y is a plugin which checks your components to see if they are compatible to a11y standards. It throws warning for everything that isn't.

#### What tests does this PR have?
None , its like linting your code

#### How can this be tested?
run `npm run docs` you should see no errors on the documentation page.
No to into the file `docs/src/customComponent.jsx` and change line 155 to remove the `role="button"` this make the page have poor accessibility.
You should now see an error thrown in the console (picture below)

#### Screenshots / Screencast
<img width="1365" alt="screen shot 2015-11-06 at 12 10 19" src="https://cloud.githubusercontent.com/assets/778942/10996907/ba8d4fcc-8480-11e5-9f7f-4d50da268130.png">

#### What gif best describes how you feel about this work?
![50 cent](https://media.giphy.com/media/IqTacCeHGTsqY/giphy.gif)

---
#### Developer Definition of Done/Quality Checklist (for PR author to complete BEFORE code review):

#### Software Engineer or Developer review:

#### Software Engineer or project guru review:
